Gemfile

```diff
@@ -5,7 +5,7 @@ gem "jekyll", "~> 3.7.3"
 group :jekyll_plugins do
   gem 'jekyll-assets'
   gem 'crds-styles', git: 'https://github.com/crdschurch/crds-styles.git', branch: 'development'
-  gem "jekyll-contentful", git: 'https://github.com/crdschurch/jekyll-contentful.git', branch: 'development'
+  gem "jekyll-contentful", git: 'https://github.com/crdschurch/jekyll-contentful.git', branch: 'master'
```

Comment:
We need to specify the version number (eg `~> 0.0.7`) here since `master` will soon contain `v1.0` of this library, which contains some breaking changes.